distributed/utils_test.py

```diff
@@ -595,6 +598,7 @@ def cluster(
     active_rpc_timeout=10,
     disconnect_timeout=20,
     scheduler_kwargs={},
+    config={},
```

Comment:
We're probably okay for now, but in general we should probably avoid having mutable default values for keyword arguments

Reply:
absolutely right. I usually do not do this but I guess the scheduler_kwargs a line above pulled me towards the dark side. we might want to integrate smth like https://pypi.org/project/flake8-mutable/ eventually